WordPress/Classes/Utility/App Configuration/AppColor.swift

```diff
@@ -91,30 +87,41 @@ struct UIAppColor {
 
 #if IS_JETPACK
     static let tint = UIColor.label
+
     static let brand = UIColor(light: CSColor.JetpackGreen.shade(.shade40), dark: CSColor.JetpackGreen.shade(.shade30))
 
     static func brand(_ shade: ColorStudioShade) -> UIColor {
         CSColor.JetpackGreen.shade(shade)
     }
+
+    static let primary = CSColor.JetpackGreen.base
+
+    static func primary(_ shade: ColorStudioShade) -> UIColor {
+        CSColor.JetpackGreen.shade(shade)
+    }
 #endif
 
 #if IS_WORDPRESS
     static let tint = brand
+
     static let brand = CSColor.WordPressBlue.base
 
     static func brand(_ shade: ColorStudioShade) -> UIColor {
         CSColor.WordPressBlue.shade(shade)
     }
+
+    static let primary = CSColor.Blue.base
```

Comment:
I haven't tested every single place where these are used, but there are a bit obscure parts, and the logic seems sound as it's simply revert to the previous behavior.